infra/base-images/base-runner/bad_build_check

```diff
@@ -64,6 +64,7 @@ function check_instrumentation {
     fi
 
     local NUMBER_OF_EDGES=$(grep -Po "INFO: Loaded [[:digit:]]+ module.*\(.*(counters|guards)\):[[:space:]]+\K[[:digit:]]+" $FUZZER_OUTPUT)
+    [[ -z "$NUMBER_OF_EDGES" ]] && return
```

Comment:
It's skipped silently, because these errors of that kind are dealt with in `check_startup_crash`, which will print helpful warnings like
```
BAD BUILD: /out/fuzz-json seems to have either startup crash or exit:
/out/fuzz-json: symbol lookup error: /out/src/shared/libsystemd-shared-239.so: undefined symbol: __asan_report_store_n
```

Reply:
Sounds reasonable! May I ask you please to add a comment like "In this case, fuzz target likely has a startup crash issue."

Reply:
@Dor1s thank you for the review. I added a comment, but I have to say I'm not very good at writing comments :-)

Reply:
Thanks a lot!